tests/test_computetools.py

```diff
@@ -0,0 +1,27 @@
+"""Tests for turbopy/computetools.py"""
+import pytest
+from turbopy.computetools import *
+
+
+@pytest.fixture
+def interpolator():
+    """Pytest fixture for basic Interpolator class"""
+    return Interpolators(Simulation({}), {"type": "Interpolator"})
+
+
+def test_interpolate1D(interpolator):
+    """Tests for turbopy.computetools.Interpolator's interpolate1D method"""
+    x = np.arange(0, 10, 1)
+    y = np.exp(x)
+    xnew = np.arange(0, 1, 0.1)
+
+    f1 = interpolator.interpolate1D(x, y)
+    f2 = interpolate.interp1d(x, y)
+    assert f1(x).all() == y.all()
```

Comment:
I don't think this is doing what you hope it is doing. You should probably use `allclose`:
```suggestion
    assert np.allclose(f1(x), y)
```